manifests/server.pp

```diff
@@ -297,6 +298,23 @@
     }
   }
 
+  if $automatic_dlo_cleanup {
+    if $facts['systemd'] {
```

Comment:
If systemd does not exist, lets try setting up a cron job to clean this up instead. For example Redhat 6 has cron but not systemd.

Reply:
done